liquidity-rewards/contracts/UniswapPoolRewards.sol

```diff
@@ -18,14 +18,14 @@ contract UniswapPoolRewards is PoolTokenWrapper, Ownable {
      * @dev We can set a reward rate per Ethereum block. 
      * Right now the average time per block is ~15 seconds, 
      * which is:
-     * 4 blocks per minute
+     * 4 blocks per minute = 0.0666... blocks/second
      * 240 blocks per hour (60 minutes)
      * 5,760 blocks per day (24 hours)
      * 172,800 blocks per month (30 days)
      * So we can use this to calculate a reward rate of 322,500 $ELIMU / 172,800 blocks/month = 1.87 $ELIMU/block 
-     * => rewardRate = 1.87 x 10e18 (token decimals) = 1870000000000000000
+     * => rewardRate = 1.87 ($ELIMU/block ) X 0.06666... (blocks/second) x 10e18 (token decimals) = 124666666666666666
```

Comment:
Can we use numbers with decimals points in the documentation to make it more readable? That way the person reading the documentation won't have to manually convert from 10e18 to decimal point numbers.